packages/dev/loaders/src/glTF/2.0/Extensions/KHR_interactivity/declarationMapper.ts

```diff
@@ -0,0 +1,1689 @@
+                output: "object",
+                inputBlockIndex: 2,
+                outputBlockIndex: 1,
+                isVariable: true,
+            },
+            {
+                input: "propertyName",
+                output: "propertyName",
+                inputBlockIndex: 0,
+                outputBlockIndex: 1,
+                isVariable: true,
+            },
+            {
+                input: "customBuildAnimation",
+                output: "generateAnimationsFunction",
+                inputBlockIndex: 0,
+                outputBlockIndex: 1,
+                isVariable: true,
+            },
+            {
+                input: "animation",
+                output: "animation",
+                inputBlockIndex: 2,
+                outputBlockIndex: 0,
+                isVariable: true,
+            },
+            {
+                input: "easingFunction",
+                output: "easingFunction",
+                inputBlockIndex: 0,
+                outputBlockIndex: 3,
+                isVariable: true,
+            },
+            {
+                input: "value_0",
+                output: "value",
+                inputBlockIndex: 0,
+                outputBlockIndex: 1,
+                isVariable: true,
+            },
+        ],
+        extraProcessor(gltfBlock, _declaration, _mapping, parser, serializedObjects) {
+            serializedObjects.forEach((serializedObject) => {
+                // check if it is the json pointer block
+                if (serializedObject.className === FlowGraphBlockNames.JsonPointerParser) {
+                    serializedObject.config = serializedObject.config || {};
+                    serializedObject.config.outputValue = true;
+                } else if (serializedObject.className === FlowGraphBlockNames.ValueInterpolation) {
+                    serializedObject.config = serializedObject.config || {};
+                    Object.keys(gltfBlock.values || []).forEach((key) => {
+                        const value = gltfBlock.values?.[key];
+                        if (key === "value" && value) {
+                            // get the type of the value
+                            const type = value.type;
+                            if (type !== undefined) {
+                                serializedObject.config.animationType = parser.arrays.types[type].flowGraphType;
+                            }
+                        }
+                    });
+                }
+            });
+            return serializedObjects;
+        },
+    },
+    "animation/start": {
+        blocks: [FlowGraphBlockNames.PlayAnimation, FlowGraphBlockNames.ArrayIndex, "KHR_interactivity/FlowGraphGLTFDataProvider"],
+        inputs: {
+            values: {
+                animation: { name: "index", gltfType: "number", toBlock: FlowGraphBlockNames.ArrayIndex },
+                speed: { name: "speed", gltfType: "number" },
+                // 60 is a const from the glTF loader
```

Comment:
Should it come from [this](https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/loaders/src/glTF/glTFFileLoader.ts#L322-L325)?

Reply:
I don't have a reference to the loader in this context. let me think if I can somehow reach it.

Reply:
Done

Reply:
Added a reference :-)